cmd/orgIdColumnPopulator/main.go

```diff
@@ -0,0 +1,197 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	_ "github.com/lib/pq"
+
+	"github.com/RedHatInsights/tenant-utils/pkg/orgIdColumnPopulator"
+
+	"github.com/RedHatInsights/tenant-utils/pkg/tenantid"
+	clowder "github.com/redhatinsights/app-common-go/pkg/api/v1"
+
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	cmd := NewRootCommand()
+	if err := cmd.Execute(); err != nil {
+		os.Exit(1)
+	}
+}
+
+func NewRootCommand() *cobra.Command {
+
+	var dbConnMetadata dbConnectionMetadata
+	var dbTable string
+	var dbAccountColumn string
+	var dbOrgIdColumn string
+	var batchSize int
+	var clowderConfig bool
+	var prometheusPushGatewayAddr string
+	var translatorServiceAddr string
+	var translatorServiceTimeout int
+	var dbOperationTimeout int
+
+	var rootCmd = &cobra.Command{
+		Use:   "orgID-column-populator",
+		Short: "Run the org_id column population util",
+		Run: func(cmd *cobra.Command, args []string) {
+
+			var err error
+
+			if clowderConfig == true {
+				dbConnMetadata, err = readDatabaseConfigFromClowder()
+				if err != nil {
+					log.Fatal(err)
+				}
+			}
+
+			connStr := buildDBConnectionString(dbConnMetadata)
+
+			db, err := initializeDBConnection(connStr)
+
+			accountNumberTranslator := buildAccountNumberTranslator(translatorServiceAddr, translatorServiceTimeout)
+
+			metricsRecorder := buildMetricsRecorder(prometheusPushGatewayAddr)
+
+			err = orgIdColumnPopulator.MapAccountToOrgId(context.TODO(), db, dbTable, dbAccountColumn, dbOrgIdColumn, dbOperationTimeout, batchSize, accountNumberTranslator, metricsRecorder)
+			if err != nil {
+				log.Fatal(err)
+			}
+		},
+	}
+
+	rootCmd.Flags().StringVarP(&dbConnMetadata.host, "db-host", "l", "localhost", "Database hostname")
```

Comment:
-h (e.g. psql uses that) might be better than -l

Reply:
Yeah, I originally tried using -h, but its already used by viper by default.  Using -h anyway appears to cause a panic from the pflag library.  :(  We could use -H.  Maybe that's a little better?